modules/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on an public DNS zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Create a `secret` with Infoblox credentials:
++
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=${INFOBLOX_USERNAME} --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=${INFOBLOX_PASSWORD}
+----
+
+. Get the `routes` to check your cluster domain:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example Output
+----
+openshift-console          console             console-openshift-console.apps.test.example.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.test.example.com                     downloads           http    edge/Redirect          None
+----
+
+.From the CLI
+
+. Create an `ExternalDNS` resource as follows:
++
+[source,yaml]
+----
+apiVersion: externaldns.olm.openshift.io/v1beta1
+kind: ExternalDNS
+metadata:
+  name: sample-infoblox
+spec:
+  provider:
+    type: Infoblox
+    infoblox:
+      credentials:
+        name: infoblox-credentials
+      gridHost: ${INFOBLOX_GRID_PUBLIC_IP}
+      wapiPort: 443
+      wapiVersion: "2.3.1"
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: test.example.com
+  source:
+    type: OpenShiftRoute
+    openshiftRouteOptions:
+      routerName: default
+----
+
+.From the Infoblox UI
```

Comment:
Is this step 4? Or possibly a verification step?

Reply:
This would be the step 4.